crates/ide/src/annotations.rs

```diff
@@ -63,18 +64,33 @@ pub(crate) fn annotations(
 
     visit_file_defs(&Semantics::new(db), file_id, &mut |def| match def {
         Either::Left(def) => {
-            let range = match def {
+            let (range, ranges_enum_variants) = match def {
                 hir::ModuleDef::Const(konst) => {
-                    konst.source(db).and_then(|node| name_range(&node, file_id))
+                    (konst.source(db).and_then(|node| name_range(&node, file_id)), vec![None])
```

Comment:
The `vec![None]` can be swapped out for just `vec![]` here instead with my other review addressed, that way we shouldn't be allocating in these branches anymore.